src/test/testType.ts

```diff
@@ -125,9 +125,15 @@ export class TestTypeImpl {
   }
 
   private _setTimeout(timeout: number) {
+    const suite = currentlyLoadingFileSuite();
+    if (suite) {
+      suite._timeout = timeout;
+      return;
+    }
+
     const testInfo = currentTestInfo();
     if (!testInfo)
-      throw new Error(`test.setTimeout() can only be called inside test or fixture`);
+      throw new Error(`test.setTimeout() can only be called inside test, describe block or fixture`);
```

Comment:
How can this error happen? If we are in a config? Maybe it should say something like `test.setTimeout() can only be called from a test file`?

Reply:
Sure.